django-testbed/frontend/src/App.js

```diff
@@ -1,16 +1,20 @@
-import { AntdUiProvider } from '@prestojs/ui-antd';
+import { getFormatterForField } from '@prestojs/ui';
+import { AntdUiProvider, getWidgetForField } from '@prestojs/ui-antd';
 import { mount, redirect, route } from 'navi';
 import React from 'react';
 import { Router, View } from 'react-navi';
 import { SWRConfig } from 'swr';
 
-import DatePicker from './DatePicker';
-
 // eslint-disable-next-line import/extensions
 import './styles/global.less?no-css-modules';
-import TimePicker from './TimePicker';
+
 import UserListView from './views/UserListView';
 
+const DatePicker = React.lazy(() => import('./DatePicker'));
+const TimePicker = React.lazy(() => import('./TimePicker'));
+const FormItemWrapper = React.lazy(() => import('@prestojs/ui-antd/build/FormItemWrapper'));
+const FormWrapper = React.lazy(() => import('@prestojs/ui-antd/build/FormWrapper'));
```

Comment:
hmm, why importing from build?

Reply:
because this is run locally not from a proper build. when we do a proper build & publish it copies stuff such that the published directory serves from the root.

Here the best we can do is import from `build`... but even that will only work if you have run a local build. I'll probably just change this to remove the `React.lazy` - i was just testing stuff and it's just the testbed so not critical